src/ViewerComponents/Loading.svelte

```diff
@@ -1,4 +1,8 @@
-<div class="loading">...searching</div>
+<script>
+    export let loading_text = '...loading';
+</script>
+
+<div class="loading">{loading_text}</div>
```

Comment:
See above i would work with the languages here. This also needs a decision which languages we want to support per default, maybe just english and german?

Reply:
See above